ortho_seq_code/plotclass.py

```diff
@@ -0,0 +1,361 @@
+import os
+import numpy as np
+from ortho_seq_code.constants_orthoseqs import *
+import matplotlib.pyplot as plt
+import click
+from ortho_seq_code.utils import create_dir_if_not_exists
+
+
+class rf1d:
+    # Initialize rf1d object
+    def __init__(
+        self, ndarray, alphbt_input, molecule="protein", phenotype=None, out_dir=None
+    ):
+        try:
+            self.x = ndarray
```

Comment:
numpy has a ndarray function, I wouldn't use that as a variable name. you could have something ndim_array

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html